python-main.js

```diff
@@ -154,6 +158,10 @@ function web_editor(config) {
     // Set the font size of the text currently displayed in the editor.
     function setFontSize(size) {
         EDITOR.ACE.setFontSize(size);
+        $("#request-repl")[0].style.fontSize = "" + size + "px";
+        if (typeof REPL !== 'undefined') {
```

Comment:
You are initialising REPL to `null` in line 128, will this ever be `underfined`?